docs/docs/submit-to-plugin-library.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Submit to Plugin Library
+---
+
+#### Publishing a plugin to the library
+
+In order to add your plugin to the [Plugin Library](/packages/), you need to publish a package to npm (learn how [here](https://docs.npmjs.com/getting-started/publishing-npm-packages)) with the [required files](#what-files-does-gatsby-look-for-in-a-plugin) and **include a `keywords` field** to `package.json` containing `gatsby` and `gatsby-plugin`.
```

Comment:
This is accidentally the wrong link -- can you change it to link to `(/plugins/)`?